pipeline_dp/mean_var_params.py

```diff
@@ -0,0 +1,132 @@
+"""DP computing of count, sum, mean, variance."""
```

Comment:
Nit: DP -> Differential privacy  (it's nice to have in general comments w/o abbrevations)